src/frontend/src/app/AppAuthenticated.tsx

```diff
@@ -26,7 +26,7 @@ const styles = {
   }
 };
 
-const AppAuthenticated: React.FC = () => {
+const AppAuthenticated = (): JSX.Element => {
```

Comment:
either add JSX.Element everywhere or don't use it anywhere. We want it to be standard